tests/lib/config/config-validator.ts

```diff
@@ -48,15 +48,15 @@ test(`if rule severity isn't valid, it should return false`, (t) => {
     t.false(valid);
 });
 
-test(`if rule schema isn't valid, it should return false`, (t) => {
-    const config = _.cloneDeep(validConfig);
+// test(`if rule schema isn't valid, it should return false`, (t) => {
```

Comment:
are we going to need it in the future? If so add some comment about when can we uncomment it, if not, remove the commented code